pkg/network/common/common_test.go

```diff
@@ -272,3 +273,62 @@ func TestParseClusterNetwork(t *testing.T) {
 		}
 	}
 }
+
+func TestParseHostSubnetEgress(t *testing.T) {
```

Comment:
by convention this should be `Test` plus the name of the function it's testing, so `TestValidateHostSubnetEgress`

Reply:
my bad, will change this as well.